core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -57,6 +57,8 @@ object ClientQuotaManagerConfig {
   val DefaultNumQuotaSamples = 11
   val DefaultQuotaWindowSizeSeconds = 1
   val MaxThrottleTimeSeconds = 30
+  // Purge sensors after 1 hour of inactivity
+  val inactiveSensorExpirationTimeSeconds  = 3600
```

Comment:
I think 1 hour should be fine. I think this is one of those configs that is unlikely to be used in practice and don't think it will add value to the operator.